terraform/application/application.tf

```diff
@@ -22,8 +22,57 @@ module "application_configuration" {
   }
 }
 
+# Run database migrations
+# https://guides.rubyonrails.org/active_record_migrations.html#preparing-the-database
+# https://github.com/ilyakatz/data-migrate
+# Terraform waits for this to complete before starting web_application and worker_application
+resource "kubernetes_job" "migrations" {
```

Comment:
Could you add the [security_context](https://github.com/DFE-Digital/terraform-modules/blob/main/aks/application/resources.tf#L169C11-L169C27) and [resources](https://github.com/DFE-Digital/terraform-modules/blob/main/aks/application/resources.tf#L103) blocks?

Reply:
module.cluster_data.configuration_map.cpu_min

Reply:
Added in f982d30529ef675d756d457c1b7ca1867a498b6b